mattermost-plugin/server/notifications.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/mattermost/focalboard/server/services/notify"
+	"github.com/mattermost/focalboard/server/services/notify/notifymentions"
+
+	pluginapi "github.com/mattermost/mattermost-plugin-api"
+
+	"github.com/mattermost/mattermost-server/v6/model"
+
+	"github.com/mattermost/mattermost-server/v6/shared/mlog"
+)
+
+const (
+	botUsername    = "boards"
+	botDisplayname = "Boards"
+	botDescription = "Created by Boards plugin."
```

Comment:
...once we have i18n in server...(just read other comment.)

Reply:
This is going to be used in the `EnsureBot` call, on the first run of the server (with the plugin), that means that 99% of the time this is going to be executed in the first run, and the only locale that you can use there is the server one, because is not bound to a user. That said, it means, that almost 99% of the time it is going to be in English anyway. We will need to add translations to the notifications, but I not sure if we need them here.